cardano_node_tests/tests/tests_plutus_v2/test_spend_datum_build.py

```diff
@@ -329,13 +329,18 @@ def test_lock_tx_big_datum(
             inline_datum=True,
         )
 
+        # We need to calculate the minimum required UTxO to be sure it's enough for a big datum
+        min_utxo = cluster.g_transaction.calculate_min_req_utxo(txouts=[script_txout]).value
```

Comment:
It is too much overhead to calculate min required UTxO in every iteration, let's just raise `amount` from `4_500_000` to e.g. `8_000_000`.